<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskAllocationWindow</class>
 <widget class="QWidget" name="taskAllocationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Partioned Scheduling</string>
  </property>
  <widget class="QGroupBox" name="taskCPUBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Task - CPU</string>
   </property>
   <widget class="QTableWidget" name="taskTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="cpuSchedBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>29</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>CPU - Scheduler</string>
   </property>
   <widget class="QTableWidget" name="cpuTable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulation Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="simulTimeEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
